pkg/lib/page.scss

```diff
@@ -213,4 +213,10 @@ body {
   .pf-v6-c-backdrop {
     --pf-t--global--background--color--backdrop--default: var(--pf-t--global--background--color--secondary--default) !important;
   }
+
+  // We set svg aliasing to precision instead of auto/optimizeSpeed to try to ensure
+  // consistent rendering between test runs.
+  svg {
+    shape-rendering: geometricprecision;
```

Comment:
With that it [still fails](https://cockpit-logs.us-east-1.linodeobjects.com/pull-21786-e1ed4f82-20250327-133707-fedora-41-other/log.html#34) :cry: 

The [documentation](https://developer.mozilla.org/en-US/docs/Web/SVG/Reference/Attribute/shape-rendering#usage_notes) actually spells this `geometricPrecision`. If that doesn't work, we can try `crispEdges`.